buildkite/resource_cluster_queue_test.go

```diff
@@ -113,7 +113,9 @@ func TestAccBuildkiteClusterQueueResource(t *testing.T) {
 				},
 				{
 					Config: configBasic(clusterName, queueKey, updatedQueueDesc),
-					Check:  ckecUpdated,
+					Check:  checkUpdated,
+					// Destroying a queue will pause its dispatch
+					ExpectNonEmptyPlan: true,
```

Comment:
So, this is kind of needed as the tear down of queues has an implicit pausing on the queue, thus setting `dispatch_paused` to `true` on delete

Reply:
This means the plan will not be empty, because the tear down causes the `false` to become `true`